### PR TITLE
gnujump: install desktop file

### PR DESCRIPTION
**Pull request by iblech**
###### Motivation for this change
Thank you, @fgaz, for maintaining gnujump! As described in #86906, I cannot actually test gnujump right now, but this pull request nevertheless adds a desktop entry for gnujump (lifted from Debian).

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/games/gnujump/default.nix [hunk 1]
```diff
@@ -11,6 +11,88 @@ stdenv.mkDerivation rec {
 
   NIX_LDFLAGS = "-lm";
 
+  postInstall = ''
+    mkdir -p $out/share/applications $out/share/pixmaps
+  
+    cat > $out/share/applications/gnujump.desktop <<EOF
+    [Desktop Entry]
```
Comment (doronbehar) [inline pkgs/games/gnujump/default.nix:18]: There's an undocumented Nix function made for this: `makeDesktopItem`.

**Comment by fgaz** [comment]
Oops, old syntax

/marvin opt-in
/status awaiting_changes

File: pkgs/games/gnujump/default.nix (hunk 1 above)
Comment (iblech) [reply pkgs/games/gnujump/default.nix]: Thank you for the pointer, back then I didn't know about makeDesktopItem. Done!

**Comment by iblech** [comment]
@SuperSandro2000 Could you review this again? I'm now using makeDesktopIcon and moved the pixmap into a separate file.

File: pkgs/games/gnujump/default.nix [hunk 2]
```diff
@@ -11,6 +20,11 @@ stdenv.mkDerivation rec {
 
   NIX_LDFLAGS = "-lm";
 
+  postInstall = ''
+    install -Dm644 -t $out/share/applications ${desktopItem}/share/applications/*
```
Comment (fgaz) [inline pkgs/games/gnujump/default.nix:24]: Use `copyDesktopItems`

Comment (iblech) [reply pkgs/games/gnujump/default.nix]: Oh, awesome, thank you for the tip! That's a nice handy tool. Fixed.

**Review by fgaz — APPROVED** [review]
Looks good

**Comment by SuperSandro2000** [comment]
This is a semi-automatic executed nixpkgs-review which is checked by a human on a best effort basis and does not build all packages (e.g. lumo, tensorflow or pytorch).
If you have any questions or problems please reach out to SuperSandro2000 on IRC.

Result of `nixpkgs-review pr 86913` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
<summary>
1 package built:
</summary>
<ul>
<li>gnujump</li>
</ul>
</details>

**Pull request merged**